### PR TITLE
Better team selection

### PR DESCRIPTION
**Pull request by YonatanGideoni**
Now instead of a normal list from which you select teams, it is a datalist where you can type team names/numbers to minimize the number of selectable teams. Addresses issue #6.

### CODE REVIEW

File: client/src/controllers/scoresheet/scoresheet_controller.js [hunk 1]
```diff
@@ -88,6 +90,7 @@ class ScoresheetController {
         if(this.scoresheet) {
             this.Scoresheet.processErrors()
         }
+        console.log(this.scoresheet.teamNumber)
```
Comment (idanstark42) [inline client/src/controllers/scoresheet/scoresheet_controller.js]: Remove this

File: client/src/controllers/scoresheet/scoresheet_controller.js [hunk 2]
```diff
@@ -76,7 +78,7 @@ class ScoresheetController {
             let self = this
             return this.teams.find(team => team.number === self.scoresheet.teamNumber).displayText
         } else {
-            return 'Missing team'
+            return 'Missing teams'
```
Comment (idanstark42) [inline client/src/controllers/scoresheet/scoresheet_controller.js]: This should stay singular. There is one team missing from the scoresheet

**Comment by YonatanGideoni** [comment]
Addressed changes, should be good now. Deleted redundant function"scoresheet.teamText()".

File: client/src/components/scoresheet/scoresheet.js [hunk 3]
```diff
@@ -1,51 +1,55 @@
...
-					<i class="fa fa-arrow-down"></i>
-				</div></il>
-				<il><div class="button" ng-click="scoresheet.reset()"><i class="fa fa-undo"></i></div></il>
-			</ul>
-		</div>
+	template: `<div class="top-bar secondary">
+	<div class="top-bar-left">
+		<ul class="menu">
+			<li><ref-identity ng-if="scoresheet.isRef"></ref-identity></li>
+			<form>
```
Comment (idanstark42) [inline client/src/components/scoresheet/scoresheet.js]: Beacuse this element no longer has `class="menu-text"`, it should be inside a `<li>` tag

File: client/src/controllers/scoresheet/scoresheet_controller.js [hunk 4]
```diff
@@ -1,3 +1,5 @@
+import scoresheet from "../../components/scoresheet/scoresheet";
```
Comment (idanstark42) [inline client/src/controllers/scoresheet/scoresheet_controller.js]: This line should be dropped

File: client/src/components/scoresheet/scoresheet.js [hunk 5]
```diff
@@ -1,51 +1,55 @@
...
+				<input type="text" list="teams" ng-model="scoresheet.scoresheet.teamNumber" onaftersave="scoresheet.processErrors()">
```
Comment (idanstark42) [inline client/src/components/scoresheet/scoresheet.js]: The attribute `onaftersave` has no meaning on an `<input>` tag. Please remove it and find another way for the validation to run every time. You can use `ng-change` perhaps.

File: client/src/components/scoresheet/scoresheet.js (hunk 3 above)
Comment (YonatanGideoni) [reply client/src/components/scoresheet/scoresheet.js]: This makes the formatting be weird and start throwing errors. Currently leaving as is.

**Pull request merged**